omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -37,6 +41,8 @@
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.multipart.MultipartFile;
 
+import javassist.bytecode.stackmap.BasicBlock.Catch;
```

Comment:
Are you using the above import anywhere?

Reply:
Removed the unused import